java/arcs/android/storage/service/IMuxedStorageService.aidl

```diff
@@ -30,6 +30,6 @@ interface IMuxedStorageService {
      * @param callback invoked whenever the storage channel responds with a
      *     message
      */
-    IStorageChannel openMuxedStorageChannel(
+    void openMuxedStorageChannel(
```

Comment:
Can you make this `oneway void` like the other methods?

(I'm not 100% sure what it does, but I believe it improves performance somehow)